software/beelzebub.yml

```
@@ -0,0 +1,14 @@
+name: "beelzebub"
+website_url: "https://beelzebub-honeypot.com/"
+source_code_url: "https://github.com/mariocandela/beelzebub"
+description: "Secure low code honeypot framework, leveraging AI for System Virtualization."
```

```suggestion
description: "Honeypot framework designed to provide a highly secure environment for detecting and analyzing cyber attacks.
```

The description should explain what the software does.

---

Thanks @nodiscc, resolved